### PR TITLE
Removes geo stats

### PR DESCRIPTION
**Pull request by grantfitzsimmons**
https://github.com/specify/specify7/pull/3948#issue-1867589353

Removes geography represented stats (like https://github.com/specify/specify7/pull/3926)

To test:

Nothing should crash because of this. Geo represented stats should just be absent from their category

### CODE REVIEW

File: specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx [hunk 1]
```diff
@@ -336,7 +329,7 @@ export const statsSpec: StatsSpec = {
           },
         },
       },
-
+      /*
```
Comment (melton-jason) [inline specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx:332]: Can you include a FIXME comment here? 
As we have done for similar feature cuts in the past.

**Review by melton-jason — APPROVED** [review]

File: specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx (hunk 1 above)
Comment (realVinayak) [reply specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx:332]: There should be one for removing taxa stats right? They both will be fixed the same time, so thought one won't be necessary here.  I'll still add one.

Comment (melton-jason) [reply specifyweb/frontend/js_src/lib/components/Statistics/StatsSpec.tsx:332]: Fair enough. Just being thorough :+1:

**Review by melton-jason — APPROVED** [review]

File: specifyweb/frontend/js_src/lib/components/Preferences/CollectionDefinitions.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const collectionPreferenceDefinitions = {
             title: 'Defines if preparation stats include total',
```
Comment (maxxxxxdlp) [inline specifyweb/frontend/js_src/lib/components/Preferences/CollectionDefinitions.tsx:29]: is this prop just for preparation stats? in that case, perhaps a more descriptive name than `showTotal` would do?
remember that we can't change pref names easily once released so have to be smart about them now.

i.e `showPreparationsTotal` or something like that that won't cause confusion and collisions in the future

**Review by maxxxxxdlp — APPROVED** [review]

**Review by CarolineDenis — APPROVED** [review]

**Review by bronwyncombs — APPROVED** [review]
Screenshot 2023-08-28 at 9 33 18 AM

File: specifyweb/frontend/js_src/lib/components/Preferences/CollectionDefinitions.tsx (hunk 2 above)
Comment (realVinayak) [reply specifyweb/frontend/js_src/lib/components/Preferences/CollectionDefinitions.tsx:29]: Resolved

**Comment by realVinayak** [comment]
To test:
Make sure preparation stats don't look off when numbers are big. For example, for herb-rbge, before these changes: 
image
Now, they should look like:
image
The alignment of number to the top, instead of center, is intentional

**Comment by grantfitzsimmons** [comment]
(Approved)

**Review by maxxxxxdlp — APPROVED** [review]

**Review by melton-jason — APPROVED** [review]

**Comment by bronwyncombs** [comment]
> also note I prefer centered rather than top-aligned

@realVinayak I second this, do it pls?

**Comment by realVinayak** [comment]
I was told to put it at top. Need to discuss that with team.

**Comment by grantfitzsimmons** [comment]
> I was told to put it at top. Need to discuss that with team.

Who said that 😭 😂

**Comment by grantfitzsimmons** [comment]
Please merge this!!

**Comment by maxxxxxdlp** [comment]
This might be the most collaborative PR in specify7's history))
8 assigned reviewers. 7 participants, 2 comitters

**Review by CarolineDenis — APPROVED** [review]

**Pull request merged**